src/cf-forms/src/atoms/label.less

```diff
@@ -2,8 +2,8 @@
     display: inline-block;
 
     &_helper {
+        .u-webfont-regular();
         color: @label-helper;
-        font-size: unit( 16px / @base-font-size-px, em );
```

Comment:
I think this will still be needed in `.a-label__heading .a-label_helper` situations.

Reply:
Fixed in https://github.com/cfpb/capital-framework/pull/614/commits/ea106aa5c784eb8f25eb0e16b7ed9c423f9a6dde